src/components/text-editor/text-editor.component.tsx

```diff
@@ -34,19 +36,53 @@ import Label from "../../__internal__/label";
 import Events from "../../__internal__/utils/helpers/events";
 import createGuid from "../../__internal__/utils/helpers/guid";
 import LabelWrapper from "./__internal__/label-wrapper";
-import { filterStyledSystemMarginProps } from "../../style/utils";
-
-const marginPropTypes = filterStyledSystemMarginProps(
-  styledSystemPropTypes.space
-);
+import {
+  BOLD,
+  ITALIC,
+  UNORDERED_LIST,
+  ORDERED_LIST,
+  InlineStyleType,
+  BlockType,
+} from "./types";
+import { LinkPreviewProps } from "../link-preview";
 
 const NUMBERS = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
-const INLINE_STYLES = ["BOLD", "ITALIC"];
-const BLOCK_TYPES = ["unordered-list-item", "ordered-list-item"];
+const INLINE_STYLES: InlineStyleType[] = [BOLD, ITALIC];
+// const BLOCK_TYPES: BlockType[] = [UNORDERED_LIST, ORDERED_LIST];
```

Comment:
if this code isn't needed we should remove it, not just comment it out

Reply:
Yep, I thought I'd deleted this already. Guess not 😄